### ISSUE TITLE
Posterior distribution for `spike` looks different than the prior distribution

### ISSUE THREAD

**Issue by FBartos** [bayesplay/bayesplay#8]
Hi Lincoln, I have a few suggestions for the package (I will open separate PR for each of them),

I would expect that a posterior distribution for spike
![image](https://user-images.githubusercontent.com/38475991/124745594-471d9280-df20-11eb-951e-8fde1021b02b.png)

would look exactly the same as the prior distribution for spike
![image](https://user-images.githubusercontent.com/38475991/124745649-513f9100-df20-11eb-9eee-81f599a6dba8.png)

The current posterior distribution kinda induces the feeling of a density surface that's incorrect.

Code:
```
data_mod <- likelihood(family = "normal", mean = 0, sd = 1)
h0_mod <- prior(family = "point", point = 0)

plot(h0_mod)
plot(extract_posterior(data_mod * h0_mod))
```

Cheers,
Frantisek

**Comment by ljcolling** [comment]
Hi Frantisek, yes, you’re right. It should look the same. Sure, you’re welcome to open a PR. Or let me know, and I’ll implement a fix.

**Comment by FBartos** [comment]
I'm actually not that familiar with S4 classes so it would be great if you would not mind implementing the fixes.
(also, as I mentioned in the remaining issues, there is no time pressure from my side)

Cheers,
Frantisek

**Comment by ljcolling** [comment]
Sure no worries. I can take care of them. I'll try and get it done soon so that I can update the CRAN version before I submit a paper describing the package.

**Comment by FBartos** [comment]
Cool, looking forward to reading it. 

Btw, let me know if you were interested in incorporating the JASP version into the paper. The JASP GUI might be an interesting alternative for people fearful of writing code. It's basically done by now - it only needs to be reviewed and merged in.

https://user-images.githubusercontent.com/38475991/125111619-68c87680-e0e6-11eb-87f8-b9881bfb4ba1.mp4

**Comment by ljcolling** [comment]
Oh awesome. Yeah that would be great.

**Comment by ljcolling** [comment]
Fixed with #12 The posterior and prior plots are now the same style

![posterior](https://user-images.githubusercontent.com/21215217/125430132-83013828-03b5-42ac-a6a7-f08c5c8721c6.png)
![prior](https://user-images.githubusercontent.com/21215217/125430138-3f4e99ed-0e90-477d-8b04-d90d83032264.png)
